parser.y

```diff
@@ -6621,15 +6621,14 @@ HintQueryType:
 HintMemoryQuota:
 	NUM Identifier
 	{
-		// May change into MB/MiB or GB/GiB
 		switch model.NewCIStr($2).L {
-		case "m":
-			$$ = getUint64FromNUM($1)
-		case "g":
-			$$ = getUint64FromNUM($1) * 1024
+		case "mb":
+			$$ = $1.(int64) * 1024 * 1024
```

Comment:
Shall we regard `MB` as the base unit? So that we can avoid multiplying `1024*1024`.

Reply:
Executor handle Byte as the base unit, I think we should sync here.

Reply:
All right.